### DISCUSSION TITLE
Setting a Custom DarkMode BG Color & Centering Article's Content

### DISCUSSION THREAD

**Discussion by ckgresla** [nunocoracao/blowfish#176] [Q&A] [answered]
Love the Blowfish theme! The documentation is absolutely brilliant and the layouts are amazing.

I had two questions; 
1. How would we set the Color manually for the background? (that is how specify the tone/color code for all backgrounds for all pages on the site) -- I dug through the docs and didn't see something directly related to this and snooped through the CSS files in the theme's `assets/css` dir
2. I would like to center the content of blog posts and leave the table of contents blank, I haven't yet been able to figure out where the styling of the main content for these webpages is being set so as to justify it in the center as opposed to the left hand side. If this is completely wrong (in terms of how the content & pages are structured) please let me know, I would appreciate any help or knowledge. 

Love the theme, thank you for making this tool publicly available!

**Comment by nunocoracao** [comment, marked as answer]
Hello @ckgresla, please take a look at [this doc](https://nunocoracao.github.io/blowfish/docs/advanced-customisation/), it explains how to override the stylesheet and get your custom CSS to override the default one. Additionally, you can always fork blowfish and re-compile the tailwind configurations. Hope this helps.

**Comment by JenPalacios** [comment]
Hey! Thanks so much for building this theme and fo such detailed documentation. Is there a way to change the background-color of the site? I changed it through a `custom.css` file. However, when doing this, the toggle of light and dark mode fails. Any particular way to accomplish this that does not involve forking blowfish and recompiling the tailwind configurations? :)

**Comment by ZhenShuo2021** [comment]
@JenPalacios try this

```css
body {
  background-color: red;
}

.dark body {
  background-color: blue;
}
```

**Reply by JenPalacios** [reply]
Ha! that was easy :) Thanks @ZhenShuo2021
